packages/ui/src/accounts/components/AccountItem/components/VestingLockItem/VestingLockItem.tsx

```diff
@@ -38,9 +37,11 @@ export const VestingLockListItem = ({
           <DetailsName>Vesting</DetailsName>
         </TitleCell>
         {!isDropped && (
-          <ValueCell>
+          <>
+            <TokenValue value={locked.add(vested)} />
             <TokenValue value={locked} />
-          </ValueCell>
+            <TokenValue value={vested} />
+          </>
```

Comment:
The current solution doesn't work:
![Screenshot from 2022-12-21 17-23-31](https://user-images.githubusercontent.com/6571453/208957655-ef93681e-a6d5-48ec-9d0c-d9aba3e97ba3.png)

This component is given the `DeriveBalancesVesting` which doesn't include `vestedClaimable` but:
Recoverable is `vestedClaimable` and what's actually locked is `locked - vested + vestedClaimable` (although there might be different way to get to these result) and finally the total should always be `locked`.

So one way to fix this is to pass `vestedClaimable` to the `VestingLockListItem`.